### PR TITLE
Update About Us pages

### PR DESCRIPTION
**Pull request by alinaboghiu**
## What does this change?

Updates the content of the about us pages in line with requests from the board. 

These were documented here: 
https://docs.google.com/document/d/1O4N2UcClX01X04yea6y6j5XRQjUHFgaqy4tkU-RPB9w/edit

The most significant changes are: 
- updating the tab names from our organisation and history to Guardian Media Group and The Scott Trust respectively 
- changing the links, images and some of the text on each page 
- moving the guardian foundation section to the Scott Trust page 

Fixes https://github.com/guardian/dotcom-rendering/issues/9594

New header:
Screenshot 2023-11-28 at 12 12 39

Changes on the landing page
Screenshot 2023-11-28 at 12 12 24

Changes on the Scott Trust page
Screenshot 2023-11-28 at 12 12 55

### CODE REVIEW

File: src/pages/organisation/index.tsx [hunk 1]
```diff
@@ -203,37 +196,31 @@ const HomePage = () => (
...
               title="GMG as a B Corporation"
               imagePath="/about/images/organisation-11.jpg"
-              linkUrl="https://www.theguardian.com/gnm-press-office/2019/oct/16/guardian-media-group-becomes-first-major-news-organisation-to-become-a-b-corporation-and-pledges-to-reach-net-zero-emissions-by-2030"
+              linkUrl="https://www.bcorporation.net/en-us/find-a-b-corp/company/guardian-media-group/"
             />
             <ResponsiveCardVariant1
               title="Positive impact and sustainability report"
               imagePath="/about/images/organisation-13.jpg"
-              linkUrl="https://www.theguardian.com/gmg/2021/aug/25/positive-impact-and-sustainability-report"
+              linkUrl="https://www.theguardian.com/gmg/2018/mar/14/corporate-reports-and-policies"
```
Comment (mxdvl) [inline src/pages/organisation/index.tsx:218]: Funny that we go from a 2021 article to a 2018 one.

File: src/pages/index.tsx [hunk 2]
```diff
@@ -137,13 +137,18 @@ const HomePage = () => (
...
+              </p>
             </>
           </InnerText>
           <div css={twoColumnResponsiveCardHolder}>
             <ResponsiveCardVariant1
-              title="A mission for journalism: an essay by our editor"
-              imagePath="/about/images/front-page-2.jpg"
-              linkUrl="https://www.theguardian.com/news/2017/nov/16/a-mission-for-journalism-in-a-time-of-crisis"
+              title="Times change but the Guardian’s values don’t: an essay by our editor"
+              imagePath="https://i.guim.co.uk/img/media/6614f6286da1a57ba2e41c953993718cbf09e473/0_0_5000_3000/master/5000.jpg?width=920&dpr=1&s=none"
```
Comment (mxdvl) [inline src/pages/index.tsx:150]: 🖼️

**Review by mxdvl — APPROVED** [review]
Top!

A couple of screenshots wouldn’t go amiss…

File: src/pages/organisation/index.tsx (hunk 1 above)
Comment (alinaboghiu) [reply src/pages/organisation/index.tsx:218]: yes I believe it is intentional

**Pull request merged**